src/models/invites-ignorer.ts

```diff
@@ -119,11 +136,10 @@ export class IgnoredInvites {
      */
     public async addSource(roomId: string): Promise<boolean> {
         // We attempt to join the room *before* calling
-        // `await this.getOrCreateSourceRooms()` to decrease the duration
+        // `await this.getSourceRooms()` to decrease the duration
         // of the racy section.
         await this.client.joinRoom(roomId);
-        // Race starts.
-        const sources = (await this.getOrCreateSourceRooms())
+        const sources = this.getSourceRooms()
             .map(room => room.roomId);
```

Comment:
why is this on two lines? It's difficult to read this way :(

Reply:
:shrug: Different codebases, different conventions. I'll adapt :)